src/main/scala/nl/knaw/dans/api/sword2/DepositProperties.scala

```diff
@@ -25,9 +26,9 @@ import scala.util.Try
 
 object DepositProperties {
   val log = LoggerFactory.getLogger(getClass)
-  case class State(label: String, description: String, timeStamp: String)
+  case class Statement(label: String, description: String, timeStamp: String)
```

Comment:
@vesaakerman:
Just to record the refactoring we just discussed in person:
- Rename the 'Statement' case class to 'DepositProperties' and add fields for all the properties in the `deposit.properties` file.
- Create and return objects of this class with the companion object's `apply` 
- Add a `getState`-method to the class that returns an enumeration value